<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>topUp</class>
 <widget class="QWidget" name="topUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="huodongBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>200</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="topup6Btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>￥6</string>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="topupbg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>960</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="topup30Btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>￥30</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="topup100Btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>￥100</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="topupdfBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义金额</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="getphoneLineEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入手机号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>phone number:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TOP UP!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selfBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>给自己充值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="othersBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>为他人充值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>choose way:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="topupLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>amount of money:</string>
   </property>
  </widget>
  <zorder>topupbg</zorder>
  <zorder>huodongBtn</zorder>
  <zorder>topup6Btn</zorder>
  <zorder>label</zorder>
  <zorder>topup30Btn</zorder>
  <zorder>topup100Btn</zorder>
  <zorder>topupdfBtn</zorder>
  <zorder>Return</zorder>
  <zorder>getphoneLineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton1</zorder>
  <zorder>selfBtn</zorder>
  <zorder>othersBtn</zorder>
  <zorder>label_3</zorder>
  <zorder>topupLineEdit</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
